LightHunters/src/Const.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+/// THIS CONST.HPP IS USED ONLY FOR THE GAME, NOT FOR THE ENGINE
+
+// Assets Paths
+#define ASSETS_PATH "assets/"
+#define FONTS_PATH ASSETS_PATH ## "fonts/"
+#define TEXTURES_PATH ASSETS_PATH ## "textures/"
+#define SOUNDS_PATH ASSETS_PATH ## "sounds/"
+
+// Font paths
+#define FONT_SPINWERARD FONTS_PATH ## "spinwerad.ttf"
+#define FONT_SPINWERARD_CAPITALIZE FONTS_PATH ## "spinweradC.ttf"
+
+
+/// <summary>
+/// Method to set stats.
+/// </summary>
+enum class SetMethod {
+	// Set the value to the given value
+	Set,
+	// Add the given value to the current value
+	Add,
+	// Multiply the current value by the given value
+	Multiply,
+};
+
+/// <summary>
+/// All actions that can be done with the keyboard.
+/// </summary>
+enum class Bindings {
+	Escape,
+	Inventory,
+	Map,
+	Skill_tree,
+	Movement,
+	Ability_1,
+	Ability_2,
+	Ability_3,
+	Ability_4,
+	Ability_5,
+	Ability_6,
+	Skip,
+};
+
+/// <summary>
+/// Layers for drawing.
+/// </summary>
+enum Layer {
+	Down, // Drawn under everything
+	Ground,
+	UpperGround,
+	Beings,
+	UpperBeings,
+	Decoration,
+	UpperDecoration,
```

Comment:
Maybe add layers for UI on top of other layers
```suggestion
	UpperDecoration,
	UiDown,
	UiBackground,
	UiLow,
	UiMid,
	UiHigh,
	UiForeground,
	UIUp,
```